modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/senna/app/App.js

```diff
@@ -537,13 +546,13 @@ class App extends EventEmitter {
 
 		this.activePath = path;
 		this.activeScreen = nextScreen;
-		this.browserPathBeforeNavigate = utils.getCurrentBrowserPathWithoutHash();
+		this.browserPathBeforeNavigate = getCurrentBrowserPathWithoutHash();
```

Comment:
This one apparently can be changed

Reply:
But not the one in https://github.com/liferay-frontend/liferay-portal/pull/489/files#diff-fd161d963fc38fff2674cb5515dad57008c6c3c72aeae2fd8e66d89ac406879cR87